.gitignore

```diff
@@ -116,3 +116,26 @@ packages/
 
 #JavaScript files
 *.jsbundle
+
+# Visual C++ cache files
+ipch/
+*.aps
+*.ncb
+*.opendb
+*.opensdf
+*.sdf
+*.cachefile
+*.VC.db
+*.VC.VC.opendb
+
+# Build results
+[Dd]ebug/
+[Dd]ebugPublic/
+[Rr]elease/
+[Rr]eleases/
+x64/
+x86/
+bld/
+[Bb]in/
+[Oo]bj/
```

Comment:
The following are already defined above:
[Oo]bj/
[Bb]in
[Dd]ebug*/
[Rr]elease*/